packages/nimble-tokens/source/styledictionary/nimble-extensions.js

```diff
@@ -80,35 +77,6 @@ StyleDictionary.registerTransformGroup({
     ]
 });
 
-// Templates and transforms to build XAML compatible token resource dictionaries
-const xamlColorTemplatePath = path.resolve(__dirname, './templates/XamlColor.template');
-console.log(`XamlColor template path: ${xamlColorTemplatePath}`);
-const xamlColorTemplate = _.template(fs.readFileSync(xamlColorTemplatePath));
-
-StyleDictionary.registerFormat({
-    name: 'xaml/XamlColor',
-    formatter: xamlColorTemplate
-});
-
-StyleDictionary.registerTransformGroup({
-    name: 'ni-xaml-color',
-    transforms: [
-        'attribute/cti',
-        'size/px',
-        'color/hex8android'
-    ]
-});
-
-// Templates and transforms to build C# token class
-const cSharpClassColorTemplatePath = path.resolve(__dirname, './templates/cSharpClassColor.template');
-console.log(`cSharpClassColor template path: ${cSharpClassColorTemplatePath}`);
-const cSharpClassColorTemplate = _.template(fs.readFileSync(cSharpClassColorTemplatePath));
-
-StyleDictionary.registerFormat({
-    name: 'cSharpClass/Color',
-    formatter: cSharpClassColorTemplate
-});
-
 StyleDictionary.registerTransform({
```

Comment:
looks like this 'color/FromRgb' transform and the 'ni-color-class' transformGroup below were just for xaml/cs so think we can nuke em

Reply:
This allowed me to remove the fast-colors dependency too 🛀